todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,34 @@
+import pymongo
+import os
+from todo_app.data.item import Item
+
+def get_items():
+    client = pymongo.MongoClient(os.getenv("MONGO_CONNECTION_STRING"))
+    database = client[os.getenv("MONGO_DATABASE_NAME")]
+    collection = database["items"]
+    
+    mongo_items = collection.find()
+
+    return [Item.from_mongo_item(mongo_item) for mongo_item in mongo_items]
+
+def add_items(title: str):
+    new_mongo_item = {
+        'task': title,
+        'status': "To Do"
+    }
+
+    client = pymongo.MongoClient(os.getenv("MONGO_CONNECTION_STRING"))
+    database = client[os.getenv("MONGO_DATABASE_NAME")]
+    collection = database["items"]
+    collection.insert_one(new_mongo_item)
+
+   
+def complete_item(mongo_id):
+    client = pymongo.MongoClient(os.getenv("MONGO_CONNECTION_STRING"))
+    database = client[os.getenv("MONGO_DATABASE_NAME")]
+    collection = database["items"]
+
+    myquery = { "_id": mongo_id }
```

Comment:
I don't think this function is quite working - the reason being that here you're comparing an "ObjectId" type (`_id`) to a string (`id`) since Mongo automatically represents its ids as ObjectIds. To resolve this, was can import the type (`from bson import ObjectId` - bson being automatically pulled in as part of pymongo) and then adjust our filter like:
```python
    myquery = { "_id": ObjectId(mongo_id) }
```